docs/source/conf.py

```diff
@@ -145,7 +145,7 @@
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.
-#html_use_smartypants = True
+html_use_smartypants = False
```

Comment:
Some of the reason is in the commit message: https://github.com/trhoden/ceph-deploy/commit/eec4c3555385d76990b7707d005575c1f309405e

But, we have a lot of section headers that start with double-dashes ``--public-network`` for example.  With smarty pants enabled (default) that gets changed to a unicode long dash.  Looks nice, but not good for code.

To work around it, a lost of place used double-back ticks to make it "pre formatted"  My problem with that was that it looked terrible. See: http://ceph.com/ceph-deploy/docs/pkg.html#install

It's dark on top of dark.  :(

So this was my way to get around that.  I wanted to get rid of the backticks to get rid of the dark on dark, but then it turned the double-dashes into long single dashes.  I don't think disable smartypants has any ill-effect (none that I could see).